run-worker.py

```diff
@@ -0,0 +1,22 @@
+"""Initializes a slim rq task queue. File taken from
+http://python-rq.org/patterns/"""
+
+import os
+import urlparse
+from redis import Redis
+from rq import Worker, Queue, Connection
+
+listen = ['high', 'default', 'low']
```

Comment:
Yeah it seems relatively non-consequential either way. I'm guessing in practice the expectation is that you place tasks in the queue according to their priority. I can't imagine that for the time being we'll have this need.